scripts/deploy.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
-./compile.sh
 
-mongod &
+pip install llama-cpp-python
 
+redis-server &
```

Comment:
To enable persistence you can run the server like this: `redis-server --save 15 1`, this will save to disk every 15 secs if at least 1 Write operation has been performed.

Source: https://hub.docker.com/_/redis

Reply:
I added persistence with an append-only file to make sure it's saved after every write!